src/main/java/com/gluonhq/NativeBaseMojo.java

```diff
@@ -197,6 +197,9 @@ private String getProjectClasspath() {
     }
 
     private List<File> getClasspathElements(MavenProject project) {
+        if (!MavenArtifactResolver.isInitialized()) {
```

Comment:
There's no need for the `isInitialized()` check, since calling `initRepositories()` will return immediately when the value is `true`. Perhaps that's a good thing to add in the javadoc for `initRepositories()` as well. The `isInitialized()` method can be removed completely too.

Reply:
Makes sense